org.sugarlabs.MazeActivity.json

```diff
@@ -0,0 +1,266 @@
+{
+    "app-id": "org.sugarlabs.MazeActivity",
+    "runtime": "org.gnome.Platform",
+    "runtime-version": "3.34",
+    "sdk": "org.gnome.Sdk",
+    "separate-locales": false,
+
+    "command": "sugarapp",
+    "finish-args": [
+        "--socket=x11",
+        "--socket=pulseaudio",
+        "--share=ipc",
+        "--device=dri",
+        "--env=SUGAR_BUNDLE_ID=org.sugarlabs.MazeActivity",
+        "--env=SUGAR_BUNDLE_PATH=/app/share/sugar/activities/Maze.activity"
+    ],
+    "cleanup": [
+        "/include",
+        "/lib/pkgconfig",
+        "/share/aclocal",
+        "/share/info",
+        "/share/man"
+    ],
+    "modules": [
+        {
+            "name": "empy",
+            "cleanup": [ "*" ],
+            "buildsystem": "simple",
+            "build-commands": [
+                "pip3 install --no-index --find-links=\"file://${PWD}\" --prefix=${FLATPAK_DEST} empy"
+            ],
+            "sources": [
+                {
+                    "type": "file",
+                    "url": "https://files.pythonhosted.org/packages/3b/95/88ed47cb7da88569a78b7d6fb9420298df7e99997810c844a924d96d3c08/empy-3.3.4.tar.gz",
+                    "sha256": "73ac49785b601479df4ea18a7c79bc1304a8a7c34c02b9472cf1206ae88f01b3"
+                }
+            ],
+            "post-install": [
+                "chmod +x /app/lib/python3.7/site-packages/em.py",
+                "mkdir -p /app/bin/",
+                "ln -s /app/lib/python3.7/site-packages/em.py /app/bin/empy"
+            ]
+        },
+        {
+            "name": "perl-XML-Simple",
+            "cleanup": [ "*" ],
+            "no-autogen": true,
+            "build-options": {
+                "no-debuginfo": true,
+                "env": "/app/lib/perl5/site_perl/5.30.0/"
```

Comment:
Have you tried building this? I am pretty sure you don't need this here and you need it in other two modules instead. Double check what I do here https://github.com/tchx84/org.sugarlabs.Chart-flathub/blob/master/org.sugarlabs.Chart.json